src/index.ts

```diff
@@ -0,0 +1,115 @@
+import designTokens from "@daikin-oss/dds-tokens/json/daikin/Light/tokens.json" with { type: "json" };
+import plugin from "tailwindcss/plugin";
+
+type TokenType =
```

Comment:
Can we get this values from actual token?

Reply:
@ukkary 
I did some research, but it turned out to be difficult.
The token type we need is defined by style-dictionary, but [it seems to accept any user-defined strings](https://github.com/amzn/style-dictionary/blob/v4.2.0/types/DesignToken.ts#L21).
Tokens Studio has [its own definition of token types](https://github.com/tokens-studio/types/blob/v0.5.1/src/constants/TokenTypes.ts), but it turns out that [some of these are converted during the process of converting to style-dictionary](https://github.com/tokens-studio/sd-transforms/blob/v1.2.9/src/preprocessors/align-types.ts#L9-L23).